<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>409</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="fuel_time">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Топливо и время</string>
       </attribute>
       <widget class="QLabel" name="labelS">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>S=</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSalt">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>42</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sзап=</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxS">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>150</number>
        </property>
        <property name="maximum">
         <number>2400</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxSalt">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="labelGt">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Топливо на полёт =</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelT">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Время полёта =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditG">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="butRasch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>85</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labelKg">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>25</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>кг</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="wight_pass">
       <attribute name="title">
        <string>Вес и загрузка</string>
       </attribute>
       <widget class="QLineEdit" name="lineEditG_2">
        <property name="geometry">
         <rect>
          <x>149</x>
          <y>17</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelGt_2">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Топливо на полёт =</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelKg_2">
        <property name="geometry">
         <rect>
          <x>206</x>
          <y>20</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>кг</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="butRaschVes">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Расчет веса</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEk">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Экипаж =</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelPr">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>92</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Б/пр =</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelTeh">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>130</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Техники =</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelZagr">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>166</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Загрузка =</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelGvzl">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>201</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Взлётный вес =</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineVzl">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>198</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelKg_Vzl">
        <property name="geometry">
         <rect>
          <x>207</x>
          <y>201</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>кг</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEk">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinPr">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>84</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinTeh">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>122</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinZagr">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Руководство</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="save"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="about"/>
    <addaction name="about_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="about_Qt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
